vars/parseStageInfo.groovy

```diff
@@ -64,16 +64,24 @@ def call(Map config = [:]) {
       echo "Could not determine target in ${env.STAGE_NAME}, defaulting to EL7"
     }
   }
+  String new_ci_target = params['CI_' +
+                                result['target'].toString().toUpperCase() +
+                                '_TARGET']
+  if (new_ci_target) {
+    result['ci_target'] = new_ci_target
```

Comment:
Line 12 still says `result` has a `target` member but doesn't mention `ci_target` member.  Is `ci_target` replacing/obsoleting `target`?  If so, why not re-use it rather than adding additional unnecessary churn to this PR and future confusion about `target` vs. `ci_target`?